cycledash/static/js/runs/components/forms.js

```diff
@@ -131,21 +145,24 @@ var NewProjectForm = React.createClass({
   },
   render: function() {
     return (
-      <form method='POST' action='/api/projects' className='project-form' ref='projectForm'>
-        <h2>
-          <button className='close' type='button'
-                  onClick={this.props.handleClose}>&times;</button>
-        </h2>
-        <h3>New Project</h3>
-
-        <TextInput label='Project Name:' name='name' placeholder='PT01234' required={true} />
-        <div className='form-group project-form-notes'>
-          <label>Notes</label>
-          <textarea className='form-control' rows='3' name='notes'
-                    placeholder='Notes etc.'></textarea>
+      <form method='POST' action='/api/projects' className='add-form' ref='projectForm'>
+        <div className='row'>
```

Comment:
Can this `row` go on the form? (Same with the other forms with a nested `div.row`).